test/functional/testCases.js

```diff
@@ -1425,7 +1425,7 @@ const testCases = [
                                 object_id: 'a',
                                 name: 'attr_a',
                                 type: 'Boolean',
-                                expression: 'a?threshold[90|tostring].max:true'
+                                expression: 'a?threshold[90|toText].max:true'
```

Comment:
```suggestion
                                expression: 'a?threshold[90|tostring].max:true'
```

As far as I understand this doesn't need to be changed (please tell me otherwise)

Reply:
Hi @fgalan , yes the change was not required. Thanks